base/uk.ac.stfc.isis.ibex.ui.synoptic.editor/src/uk/ac/stfc/isis/ibex/ui/synoptic/editor/dialogs/SaveSynopticDialog.java

```diff
@@ -39,28 +37,30 @@
 import org.eclipse.swt.widgets.Shell;
 import org.eclipse.swt.widgets.Text;
 
-import com.google.common.base.Predicate;
-import com.google.common.collect.Iterables;
-
+/**
+ * The dialog used to save a synoptic.
+ */
```

Comment:
This dialog can be made smaller now that we never display error messages below the text entry field. It looks a little strange with such a big empty space.